client/src/app/site/assignments/models/view-assignment.ts

```diff
@@ -0,0 +1,53 @@
+import { BaseViewModel } from '../../base/base-view-model';
+import { Assignment } from '../../../shared/models/assignments/assignment';
+import { Tag } from '../../../shared/models/core/tag';
+import { User } from '../../../shared/models/users/user';
+import { Item } from '../../../shared/models/agenda/item';
+
+export class ViewAssignment extends BaseViewModel {
+    private _assignment: Assignment;
+    private _relatedUser: User[];
+    private _agendaItem: Item;
+    private _tags: Tag[];
+
+    public get assignment(): Assignment {
+        return this._assignment;
+    }
+
+    public get relatedUser(): User[] {
+        return this._relatedUser;
+    }
+
+    public get agendaItem(): Item {
+        return this._agendaItem;
+    }
+
+    public get tags(): Tag[] {
+        return this._tags;
+    }
+
+    /**
+     * unknown where the identifier to the phase is get
+     */
+    public get phase(): number {
+        return this.assignment ? this.assignment.phase : null;
+    }
+
+    public get candidateNumber(): number {
```

Comment:
Isn't it count instead of number?

Reply:
English does not really differentiate between number as a quantity and number as a digit.
I would suggest "amount" since we really want to get a positive quantity of candidates

Reply:
ok.